pkg/controllers/routing/network_routes_controller.go

```diff
@@ -1366,9 +1368,17 @@ func NewNetworkRoutingController(clientset kubernetes.Interface,
 	}
 	nrc.isIPv6Capable = len(nrc.nodeIPv6Addrs) > 0
 
-	if kubeRouterConfig.RouterID != "" {
+	switch {
+	case kubeRouterConfig.RouterID == "generate":
```

Comment:
Can you add some information to the documentation about this option for setting the router ID?

Probably needs to go here: https://github.com/cloudnativelabs/kube-router/blob/master/pkg/options/options.go#L201 (this will necessitate updating the options here: https://github.com/cloudnativelabs/kube-router/blob/master/docs/user-guide.md)

Reply:
Done